placement/algo/mirrored.go

```diff
@@ -74,17 +74,7 @@ func (a mirroredAlgorithm) InitialPlacement(
 		return nil, err
 	}
 
-	p, err := placementFromMirror(mirrorPlacement, instances, rf)
-	if err != nil {
-		return nil, err
-	}
-
-	// NB(cw): Do not validate shards for initial placement.
```

Comment:
Do we no longer need this? I'm also not sure why we need this in the first place.. Shouldn't all shards always be marked as available in the initial placement?

Reply:
We no longer need this as the sharded algo is already marking them as available

Reply:
So it's not needed before?